### ISSUE TITLE
ZipCompleter: problems

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Risehat/freemedforms#235]
```
State/Province is now available in ZipCodes datapacks but is not already saved 
into patientbase.
```

Original issue reported on code.google.com by `eric.mae...@gmail.com` on 7 Dec 2012 at 10:19

**Comment by GoogleCodeExporter** [comment]
```
Done, one view & database

BUT

the ZipCompleter does not handle the Provinces
```

Original comment by `eric.mae...@gmail.com` on 7 Dec 2012 at 11:15
- Changed title: **ZipCompleter: manage State/Province**

**Comment by GoogleCodeExporter** [comment]
```
- completer does not show on City field
- when setting the currentIndex -> filtering city/zip, but the state/province 
stays unset

```

Original comment by `eric.mae...@gmail.com` on 12 Dec 2012 at 4:13
- Changed title: **ZipCompleter: problems**
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
Original comment by `christian.a.reiter@gmail.com` on 12 Dec 2012 at 5:24
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
When entering some chars into a zipcompleter field, nothing happens. This is 
because the SELECT is going into /dev/null:
SQL Error: 
* no such table: IMPORT
* SELECT `ZIP`, `CITY`, `ADMIN_NAME1` FROM `IMPORT` WHERE `COUNTRY`="AT" AND 
`CITY` LIKE "wer%" ORDER BY `CITY` ASC LIMIT 0, 50" 

Why is ZipCompleters querying the IMPORT table??  why not the ZIPS? I thought 
the IMPORT table is temporary?
My zipcodes.db sqlite file only has a ZIPS table.
```

Original comment by `christian.a.reiter@gmail.com` on 12 Dec 2012 at 10:59

**Comment by GoogleCodeExporter** [comment]
```
Recreate your zipcode database with FTB. I kept ZIPS for compatibility with old 
version and IMPORT for the new versions...
```

Original comment by `eric.mae...@gmail.com` on 12 Dec 2012 at 11:02

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 86bbd19a2505.
```

Original comment by `eric.mae...@gmail.com` on 20 Dec 2012 at 8:06
- Changed state: **Fixed**
